### DISCUSSION TITLE
Can't apply to the student pack.

### DISCUSSION THREAD

**Discussion by cauvang32** [community/community#140995] [GitHub Education] [answered]
What error I got:
- You appear not to be near any campus location for the school you have selecte.....

What I have tried:
- Stand near my school (yeah I register when I at school)
- Used cellcular data and wifi (tried  both)
- Tried register at home where far from my campus for 2 km.
- Disabled VPN, Proxy and anything else.

**Comment by davevad93** [comment, marked as answer]
Hi @cauvang32 ,

It's possible that GitHub may have incorrect information about your school's location.

Try contacting support by opening a ticket [here](https://support.github.com/contact/education), select "`My selected school has incorrect or incomplete information e.g. domains or campus location`" and explain your issue.
